kategory/src/main/kotlin/kategory/instances/IdInstances.kt

```diff
@@ -41,3 +41,14 @@ interface IdInstances :
             fa.ev().value
 
 }
+
+object IdSemigroupK : SemigroupK<Id.F>, GlobalInstance<SemigroupK<Id.F>>() {
+    override fun <A> combineK(x: HK<Id.F, A>, y: HK<Id.F, A>): HK<Id.F, A> = y
+}
+
+class IdMonoidK(val a: Any) : MonoidK<Id.F>, GlobalInstance<MonoidK<Id.F>>() {
```

Comment:
Brief summary: I couldn't generify the Id.F type there, since the type A is defined on methods, and cannot be ready on construction. You end up in a loop where you can raise a value of A on the method, since the A is being provided there and you can't ask for an Applicative of A in the class, since the A is still not defined. If you have any idea, just let me know!